neurovault/apps/statmaps/views.py

```diff
@@ -1196,3 +1197,40 @@ class MyCollectionsJson(PublicCollectionsJson):
 
     def get_initial_queryset(self):
         return get_objects_for_user(self.request.user, 'statmaps.change_collection')
+
+
+def download_collection(request, cid):
+    # From https://stackoverflow.com/questions/67454/serving-dynamically-generated-zip-archives-in-django
+
+    # Files (local path) to put in the .zip
+    collection = get_collection(cid, request)
+    filenames = [ img.file.path for img in collection.basecollectionitem_set.all()]
```

Comment:
Depending on the configuration of production, this might be 

`filenames = [ img.file.url for img in collection.basecollectionitem_set.all()]` , not sure, but it works as it is now for me.